src/meta-srv/src/bootstrap.rs

```diff
@@ -229,7 +233,15 @@ pub async fn metasrv_builder(
             let kv_backend = PgStore::with_pg_client(pg_client)
                 .await
                 .context(error::KvBackendSnafu)?;
-            (kv_backend, None)
+            let election_client = create_postgres_client(opts).await?;
+            let election = PgElection::with_pg_client(
+                opts.server_addr.clone(),
+                election_client,
+                opts.store_key_prefix.clone(),
+                CANDIDATE_LEASE_SECS,
```

Comment:
> I think the two options (CANDIDATE_LEASE_SECS and KEEP_ALIVE_INTERVAL_SECS) should control the entire election backend.

Yes it is. The newly added param is just for unit test since it's too long for tests to wait for expiration.

Reply:
Got it.